### ISSUE TITLE
Keras to MDF

### ISSUE THREAD

**Issue by Onoyiza** [ModECI/MDF#491]
The current Keras to MDF conversion pipeline only works when the Keras model contains an Input Layer and/ Flatten Layers and / Dense Layers. It uses a function that checks the types of layers in the Keras model and maps out the equivalent node representation in an MDF graph and model. Frequently used layers like Dropout, BatchNormalization for any network and Conv2D, Conv3D, and Maxpooling for convolutional networks do not currently work using the pipeline. 

Solutions that are currently being considered:
- Get a working equivalent node representation of the layers in MDF using Onnx operators and functions
- Using the tf2onnx tool for the conversion (Has support for commonly used layers but not all layers)

Resources to foster solution:
[Supported Operations for Tensorflow/Keras to Onnx](https://github.com/onnx/tensorflow-onnx/blob/main/support_status.md)
[Common Conversion Issues Encoutered](https://github.com/onnx/tensorflow-onnx/blob/main/Troubleshooting.md)
[Sample Tutorial using Resnet model from Keras converted to Onnx](https://github.com/onnx/tensorflow-onnx/blob/main/tutorials/keras-resnet50.ipynb)
[Repo for Tensorflow to Onnx Conversion](https://github.com/onnx/tensorflow-onnx)

**Comment by pgleeson** [comment]
Thanks for adding details on this @Onoyiza!
